src/services/DeviceService.js

```diff
@@ -0,0 +1,24 @@
+import client from './AxiosClientService';
+
+import DeviceModel from '../models/DeviceModel';
+import DevicesStore from '../stores/DevicesStore';
+
+class Device {
```

Comment:
Naming convention: service should have `Service` suffix in class name (`DeviceService`)